### ISSUE TITLE
Braze integration needs update

### ISSUE THREAD

**Issue by WojciechTrzasko** [mParticle/mparticle-apple-sdk#91]
Hi,

A few days ago Braze identified an issue with Braze iOS SDK that could potentially result in data not being sent by the SDK to the Braze servers when the SDK is not properly configured. They released a fixed version and sent emails to their clients with a request to update.

The 3.17.0 version contains a few minor changes that break the mParticle's integration library, which always use the newest version as a dependency.

The main problem is that with the new version Braze team decided to remove the possibility of misconfiguring the SDK and therefore the `ABKAppboyEndpointDelegate` and `ABKAppboyEndpointDelegateKey` are no longer available (both used by mParticle's integration library).

Is there any plan to fix this issue and release the newest version of mParticle's integration library?

Thank you for any answer.

**Comment by Palbrevik** [comment]
+1

**Comment by WojciechTrzasko** [comment]
Hi @samdozor,
Thanks for releasing the fix :)

Can you please tell me what is the status of release? I've seen that after merging the fix, you set Podfile back to the version to 7.10.5 (so the previous one). What was the reason? Is this fix breaking something?

**Comment by samdozor** [comment]
hey @WojciechTrzasko we'll have to look at that bot, it's definitely did something weird. What happened was:
- Braze released 3.17.0 and that broke the kit. We updated it and manually released it as 7.10.5. Core SDK was left at 7.10.4.
- Braze released 3.18.0 very shortly after with *another* interface change. We updated it again and released 7.10.6
- Coincidentally we released 7.10.5 of the core SDK for unrelated updated. The bot went ahead and updated the appboy repo to match. This was just in git and didn't actually affect the cocoapods release.

So - 7.10.5 which is compatible with Braze 3.17 and 7.10.6 which is compatible with Braze 3.18 are both released. We'll clean up git in the Braze repo but otherwise there should be no effect.

Closing this issue - let me know if you're still experiencing incompatibility. Thanks!
